src/math/triangular_number.rs

```diff
@@ -0,0 +1,30 @@
+// Triangular Numbers: Function to the Nth Triangular Number
+// Wikipedia Reference : https://en.wikipedia.org/wiki/Triangular_number
+use num_bigint::BigUint;
+pub fn triangular_number(n: u64) -> BigUint {
+    let m: BigUint = ((n | 1) * ((n + 1) / 2)).into();
+    m
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    macro_rules! test_triangular_number {
+        ($($name:ident: $inputs:expr,)*) => {
+        $(
+            #[test]
+            fn $name() {
+                let (n, expected) =$inputs;
+                assert_eq!(triangular_number(n), expected);
+            }
+        )*
+        }
+    }
+
+    test_triangular_number! {
+        input_5: (6, BigUint::from(21u32)),
+        input_9: (10, BigUint::from(55u32)),
+        input_10: (100, BigUint::from(5050u32)),
```

Comment:
What is the logic behind the names of the test cases?

Please add a text case with _odd_ input and the _first ones_ like:
```rust
        input_0: (0, BigUint::from(0u32)),
        input_1: (1, BigUint::from(1u32)),
```